src/lib/flex/layout-gap/layout-gap.ts

```diff
@@ -200,14 +200,11 @@ export class LayoutGapDirective extends BaseDirective2 implements AfterContentIn
     }
   }
 
-  /**
-   * Quick accessor to the current HTMLElement's `display` style
-   * Note: this allows us to preserve the original style
-   * and optional restore it when the mediaQueries deactivate
-   */
-  protected getDisplayStyle(source: HTMLElement = this.nativeElement): string {
-    const query = 'display';
-    return this.styler.lookupStyle(source, query);
+  /** Determine if an element will show or hide based on current activation */
+  protected willDisplay(source: HTMLElement): boolean {
+    const value = this.marshal.getValue(source, 'show-hide');
```

Comment:
How about this for better clarity:

```ts
const value = this.marshal.getValue(source, 'show-hide');
if ( value == '' || value == undefined) {
  value = this.styleUtils.lookupStyle(source, 'display') !== 'none';
}
return value;
```

Reply:
I don't think the value gain is great in terms of clarity: the value returned from the marshaller will never be undefined.